maud/Cargo.toml

```
@@ -26,6 +26,7 @@ futures-util = { version = "0.3.0", optional = true, default-features = false }
 actix-web-dep = { package = "actix-web", version = "4", optional = true, default-features = false }
 tide = { version = "0.16.0", optional = true, default-features = false }
 axum-core = { version = "0.1", optional = true }
+poem = { version = "~1", optional = true }
```

Any particular reason for the `~` dependency? I'd rather avoid these uncommon constructs unless there's a specific reason to use them.

```suggestion
poem = { version = "1.3", optional = true }
```